libkbfs/key_manager.go

```diff
@@ -606,6 +608,8 @@ func (km *KeyManagerStandard) Rekey(ctx context.Context, md *RootMetadata, promp
 
 		wRemoved := km.usersWithRemovedDevices(ctx, md.TlfID(), wDkim, wKeys)
 		rRemoved := km.usersWithRemovedDevices(ctx, md.TlfID(), rDkim, rKeys)
+		// TODO: This is incorrectly true if we're only
+		// promoting readers.
```

Comment:
nit: ditto. may want to reference other location that relies on this re: my other nit.